package.json

```diff
@@ -24,7 +24,8 @@
     "aws-bootstrap": "node cdk/bootstrap.js",
     "aws-destroy": "node cdk/destroy.js",
     "comb": "jsoncomb --normalise \"./locales/*.json\"",
-    "cypress:open": "node ./node_modules/.bin/cypress open"
+    "cypress:open": "node ./node_modules/.bin/cypress open",
+    "postinstall": "patch-package"
```

Comment:
If you're confused about why this is here, as it's not directly related to this change.

`patch-package` is required for the NodeStarter route generator to function correctly. Not sure why it was removed previously, just restoring 'cause I wanted to use it.